app/src/main/java/swati4star/createpdf/util/PDFUtils.java

```diff
@@ -620,7 +620,7 @@ private boolean isInputValid(String path, String[] ranges) {
      * ERROR_PAGE_NUMBER if range greater than max number of pages
      * ERROR_PAGE_RANGE if range is invalid like 9-4
      */
-    private int checkRangeValidity(int numOfPages, String[] ranges) {
+    public static int checkRangeValidity(int numOfPages, String[] ranges) {
```

Comment:
Do we need to make it static?

Reply:
I think making it static would make things easier for him. In case of it being non-static, he'll have to create an instance of PDFUtils class which expects a context in the constructor. As a result he'll have to mock an activity as well. So making it static, solves that issue.